lib/engine/game/g_18_usa/step/loan.rb

```diff
@@ -1,12 +1,20 @@
 # frozen_string_literal: true
 
 require_relative '../../g_1817/step/loan'
+require_relative 'scrap_train_module'
 
 module Engine
   module Game
     module G18USA
       module Step
         class Loan < G1817::Step::Loan
+          include ScrapTrainModule
```

Comment:
Why is scrap train needed here?

Reply:
By putting scrap train here, there is only one blocking step in the18USA operating round instead of two